stories/vrc-stories/ContentModal.js

```diff
@@ -1,37 +1,101 @@
-/* eslint-disable import/no-extraneous-dependencies */
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { text, object } from '@storybook/addon-knobs';
 import styles from 'lib/styles/build/js/design_tokens';
-import { Button, ContentModal, Tabs, Tab, TabPanel, TabList, Icon } from '@';
+import {
+  Button,
+  ContentModal,
+  Tabs,
+  Tab,
+  TabPanel,
+  TabList,
+  Icon,
+  Table,
+  TR,
+  TH,
+  TD,
+  THead,
+  TBody,
+} from '@';
+
+const sample = [
+  { event: 'Activation Started', details: 'by user@domain.com', date: '2021-07-03' },
+  { event: 'Secure File Encrypted', details: 'by user@domain.com', date: '2020-05-02' },
+  { event: 'Secure Email Sent', details: 'by user@domain.com', date: '2019-07-01' },
+];
+const data = new Array(10).fill(sample).flat();
+
+const getContentHeight = (title, subtitle, offset = 0) => {
+  let varHeight = offset;
+  const subtitleHeight = 60;
+  const titleHeight = 49;
+  if (title && !subtitle) {
+    varHeight -= subtitleHeight;
+  } else if (!title && subtitle) {
+    varHeight -= titleHeight;
+  } else if (!title && !subtitle) {
+    varHeight -= titleHeight + subtitleHeight;
+  }
+  const fullHeaderHeight = 72;
+  const bottomModalMargin = 36;
+  const bottomContentPadding = 24;
+  const titleAndSubtitle = 133;
+  const overlayTopMargin = 70;
+  const overlayBottomMargin = 15;
+  const nonContentSpace =
+    fullHeaderHeight +
+    bottomModalMargin +
+    bottomContentPadding +
+    titleAndSubtitle +
+    overlayTopMargin +
+    overlayBottomMargin;
+  return -varHeight - nonContentSpace;
+};
```

Comment:
We dont need that. We should move all this to css. So what you want to do is set container with `overflow auto` that way it will add scroll only if content is overflowing it. Also some trics from here can help
https://css-tricks.com/snippets/css/a-guide-to-flexbox/

If youll have any problems you can call me anytime and we will fix it as css